packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2782,6 +2782,60 @@ moduleFor('Components test: curly components', class extends RenderingTest {
     }, /didInitAttrs called/);
   }
 
+  ['@test didInitAttrs is invoked before finishChains'](assert) {
+    expectDeprecation(/didInitAttrs called/);
+
+    let component;
```

Comment:
I would prefer to avoid rendering entirely, the timings this is testing occurs well before rendering.

If there is a better file to put more focused tests, please direct me to it.